static/src/stylesheets/module/nav/_brand-bar.scss

```diff
@@ -98,6 +100,22 @@
     }
 }
 
+.brand-bar__item__badge {
+    // TODO: Palette?
+    background: lighten(colour(guardian-brand), 43%);
```

Comment:
Where can I go to visualise our colour palette? I'm a bit lost with it all, without some kind of visualisation.